chrome/content/FolderPaneSwitcher.js

```diff
@@ -55,111 +66,124 @@ var FolderPaneSwitcher = {
         }
       }
       if (!this.views[m] || (this.views[m]['menu_enabled'] &&
-                             this.views[m]['arrows_enabled'])) {
+        this.views[m]['arrows_enabled'])) {
         prevMode = m;
       }
     }
     if (prevMode) {
       gFolderTreeView.mode = prevMode;
     }
   },
-    
-  goForwardView: function() {
-    var currentMode = gFolderTreeView.mode;
+
+  goForwardView: function (event) {
+  
+  console.log("goforw");
+  console.log("actmodes", gFolderTreeView._activeModes, gFolderTreeView._modes);
+  //gFolderTreeView.activeModes = "favorite";//gFolderTreeView._modes["favorite"];
+  console.log("actmodes", gFolderTreeView._activeModes, gFolderTreeView.activeModes[0]);
+      var currentMode = gFolderTreeView.activeModes[gFolderTreeView.activeModes.length-1];
```

Comment:
Please properly format your code. Visual Studio Code actually has a function for that. Right mouse click inside your code -> Format Document will properly align your code.